.github/workflows/scripts/e2e-verify.sh

```diff
@@ -34,10 +37,19 @@ e2e_verify_predicate_invocation_environment "$ATTESTATION" "github_event_name" "
 e2e_verify_predicate_invocation_environment "$ATTESTATION" "github_ref" "$GITHUB_REF"
 e2e_verify_predicate_invocation_environment "$ATTESTATION" "github_ref_type" "$GITHUB_REF_TYPE"
 
-if [[ -z "$LDFLAGS" ]]; then
-    e2e_verify_predicate_buildConfig_command "$ATTESTATION" "[\"build\",\"-mod=vendor\",\"-trimpath\",\"-tags=netgo\",\"-o\",\"$BINARY\",\"./pr-e2e/go/main.go\"]"
+# First compilation step is vendoring
+e2e_verify_predicate_buildConfig_step_command "0" "$ATTESTATION" "[\"mod\",\"vendor\"]"
+echo "[\"PWD=$PWD/builders/go/e2e-presubmits\"]"
+echo "PWD: $PWD"
+echo -n "$ATTESTATION" | jq '.predicate.buildConfig.steps[0].env | sort'
```

Comment:
Is this debug output or do you want to keep this?

Reply:
I'm going to remove this. Sorry I've been searching for a bug which I just found ^^

Reply:
Removed.